drivers/overlay/overlay.go

```diff
@@ -309,7 +310,9 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 	if err != nil {
 		return nil, err
 	}
-	if fsName, ok := graphdriver.FsNames[fsMagic]; ok {
+	if fsName, ok := graphdriver.FsNames[fsMagic]; !ok {
+		return nil, errors.Wrapf(graphdriver.ErrIncompatibleFS, "filesystem type %#x reported for %s is not supported with 'overlay'", fsMagic, filepath.Dir(home))
+	} else {
```

Comment:
Just remove the else { } block

and only have
backingFs=fsName

Should fix the lint complaint.